VCFv4.3.tex

```diff
@@ -43,6 +43,11 @@ \section{The VCF specification}
 In order to ensure interoperability across platforms, VCF compliant implementations must support
 both LF (\texttt{\textbackslash n}) and CR+LF (\texttt{\textbackslash r\textbackslash n}) newline conventions.  
 
+Unless explicitly specified elsewhere, all fields are encoded using 7-bit
```

Comment:
As mentioned on the call, vcf is already explicitly [UTF-8](https://github.com/jkbonfield/hts-specs/blob/3b6d69a56509eeb1e75b8686d81b5b67043652c3/VCFv4.3.tex#L83-L93).